### PR TITLE
[enable_counters.py] Convert to Python 3

### PR DESCRIPTION
**Pull request by jleveque**
**- Why I did it**

As part of moving all SONiC code from Python 2 (no longer supported) to Python 3

**- How I did it**

- Convert enable_counters.py script to Python 3
- Reorganize imports per PEP8 standard
- Two blank lines precede functions per PEP8 standard

**- How to verify it**

Ensure enable_counters.py script still functions correctly

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

### CODE REVIEW

**Comment by jleveque** [comment]
Retest broadcom please

**Comment by jleveque** [comment]
Retest vsimage please

**Comment by jleveque** [comment]
Retest vsimage please

File: dockers/docker-orchagent/enable_counters.py [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+
 def enable_counter_group(db, name):
     info = {}
     info['FLEX_COUNTER_STATUS'] = 'enable'
     db.mod_entry("FLEX_COUNTER_TABLE", name, info)
 
+
 def enable_counters():
-    db = swsssdk.ConfigDBConnector()
+    db = swsssdk.ConfigDBConnector(decode_responses=True)
```
Comment (qiluo-msft) [inline dockers/docker-orchagent/enable_counters.py:15]: >decode_responses=True [](start = 35, length = 21)

It is actually not required.

Comment (jleveque) [reply dockers/docker-orchagent/enable_counters.py]: Thanks. Removed.

**Review by pavel-shirshov — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**